frontend/src/components/panels/LayerTree.vue

```diff
@@ -25,22 +25,32 @@
 							:title="layer.visible ? 'Visible' : 'Hidden'"
 						/>
 					</div>
+					<button
+						v-if="layer.layer_type === LayerType.Folder"
+						class="node-connector"
+						:class="{ expanded: layer.layer_data.expanded }"
```

Comment:
This should be updated when the folder is expanded or collapsed (currently it is only updated when the layer tree modified again - by modifying the selection or by drawing or deleting a layer.

Reply:
fixed